File: FitzRepresentacoes/Models/ClienteModel.cs
using System.ComponentModel.DataAnnotations;

namespace FitzRepresentacoes.Models
{
    public class ClienteModel
    {
        [Key]
        public int id { get; set; }
        public string Nome { get; set; }
        public string Documento { get; set; }
        public CidadeModel Cidade { get; set; }
        public string Rua { get; set; }
        public string Bairro { get; set; }
        public string Cep { get; set; }
        [DataType(DataType.Date)]
        public DateTime dthNascimeto { get; set; }
        public List<PedidoModel> Pedidos { get; set; }
    }
}

File: FitzRepresentacoes/Scripts/Models/ClienteModel.ts
import { CidadeModel } from "./CidadeModel.js"

export interface ClienteModel {
    id: number,
    nome: string,
    documento: string,
    cidade: CidadeModel,
    rua: string,
    bairro: string,
    cep: string,
    dthNascimento: Date,
    email: string,
    telefone: string

}